### PR TITLE
Add support for Windows

### PR DESCRIPTION
**Pull request by e-rk**
Added .bat script and modified extcap to run on Windows.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,12 +21,19 @@ To start using the sniffer, you must flash the firmware, install the script, and
 nrfjprog -f nrf52 --program nrf802154_sniffer/nrf802154_sniffer.hex --chiperase -r
 ```
 
-### Install extcap script
+### Install extcap script on Linux
```
Comment (hubertmis) [inline README.md]: I would keep `### Install extcap script` and add `#### Linux` and `#### Windows` as subsections.

File: README.md [hunk 2]
```diff
@@ -22,11 +22,20 @@ nrfjprog -f nrf52 --program nrf802154_sniffer/nrf802154_sniffer.hex --chiperase
...
 ```
 sudo cp nrf802154_sniffer/nrf802154_sniffer.py /usr/lib/x86_64-linux-gnu/wireshark/extcap/
 ```
 
+#### Windows
+Copy both .bat and .py scripts from nrf802154_sniffer to the appropriate directory:
+```
+<wireshark-install-dir>\extcap\
+```
+Ensure that Python directory is included your PATH system environment variable.
```
Comment (LuDuda) [inline README.md]: in your

File: README.md [hunk 3]
```diff
@@ -22,11 +22,20 @@ nrfjprog -f nrf52 --program nrf802154_sniffer/nrf802154_sniffer.hex --chiperase
...
 ### Install extcap script
+
+#### Linux
 Copy the provided script to the appropriate directory:
 ```
 sudo cp nrf802154_sniffer/nrf802154_sniffer.py /usr/lib/x86_64-linux-gnu/wireshark/extcap/
 ```
 
+#### Windows
+Copy both .bat and .py scripts from nrf802154_sniffer to the appropriate directory:
```
Comment (LuDuda) [inline README.md]: not: let say nrf802154_sniffer.bat and nrf802154_sniffer.py instead of pointing to extension itself.

File: README.md [hunk 4]
```diff
@@ -2,9 +2,9 @@
 
 This repository contains firmware and Wireshark extcap script that can be used with nRF52840 chip as an 802.15.4 sniffer.
 
-__Note:__ this project is experimental. There is __no Windows support__ in the current version.
-
-The software provided has been tested with the nRF52840-DK board and Wireshark 2.4.5 from the official Ubuntu 18.04 package.
+The software provided has been tested with the nRF52840-DK board and the following operating systems:
```
Comment (stig-bjorlykke) [inline README.md]: I have also tested this on macOS Mojave (Wireshark 2.6.4).

File: README.md [hunk 5]
```diff
@@ -22,11 +22,20 @@ nrfjprog -f nrf52 --program nrf802154_sniffer/nrf802154_sniffer.hex --chiperase
 ```
 
 ### Install extcap script
+
+#### Linux
 Copy the provided script to the appropriate directory:
 ```
 sudo cp nrf802154_sniffer/nrf802154_sniffer.py /usr/lib/x86_64-linux-gnu/wireshark/extcap/
```
Comment (stig-bjorlykke) [inline README.md]: The correct destination for the extcap utility can be found in "About Wireshark" -> "Folders" -> "Extcap path" for each OS.

File: README.md [hunk 6]
```diff
@@ -2,9 +2,9 @@
 
 This repository contains firmware and Wireshark extcap script that can be used with nRF52840 chip as an 802.15.4 sniffer.
 
-__Note:__ this project is experimental. There is __no Windows support__ in the current version.
```
Comment (hubertmis) [inline README.md]: I meant to keep '__Note:__ this project is experimental.' and remove the other sentence.

**Comment by e-rk** [comment]
Thank you for your suggestions. I have updated the readme.

**Pull request merged**